certbot-consul/setup.py

```diff
@@ -0,0 +1,22 @@
+"""Setup script for certbot-consul"""
+
+from setuptools import find_packages, setup
+
+version = '1.22.0.dev0'
+
+setup(
+    name='certbot-consul',
+    version=version,
+    packages=find_packages(),
+    python_requires='>=3.6',
```

Comment:
But `@dataclass` is available only from 3.7?

Reply:
The `dataclasses` library is backported to 3.6: https://pypi.org/project/dataclasses/ That library is also listed in [`requirements.txt`](https://github.com/certbot/certbot/blob/86406ab63aebf463ca4aa0381a55ddeb91231cd2/tools/requirements.txt#L40) even though it isn't used in the certbot code itself. But I recon it shouldn't be a problem if a plugin would use it?

Reply:
I hadn't noticed this.